securing_apps/topics/oidc/java/jboss-adapter.adoc

```diff
@@ -29,7 +29,11 @@ Adapters are available as a separate archive depending on what server version yo
 NOTE: {appserver_name} should be running when you install the adapter. If you have either one running, you must stop it before installing and then restart it after installation is complete.
 
 ifeval::[{project_community}==true]
-Install on WildFly 9, 10 or 11:
+
+NOTE: WildFly adapters on version 11 or older are deprecated. They may still work, but we cannot guarantee to regularly test them
```

Comment:
_I'd rather not mention version number explicitly. WDYT about something about the following formulation?_

We only test and maintain adapter with the most recent version of WildlFly available upon the release. Once new version of WildFly is released, the current adapters become deprecated and support for them will be removed after next WildFly release.

Reply:
I like that!